.github/.wordlist.txt

```diff
@@ -0,0 +1,9 @@
+anuket
+Baraque
+Kubernetes
+Lakelse
+OpenStack
+sexualized
```

Comment:
I do not know what is the use of this file, but this word is a bit strange here. ..

Reply:
It's part of the [code of conduct document](https://github.com/cntt-n/CNTT/blob/master/doc/CODE_OF_CONDUCT.rst)

Reply:
That is clear. I just do not know if `.worldlist.txt` contains words what we would like to allow, like Baraque or words what we would like to avoid.

Reply:
Yes, that list contains the additional words not included in the standard English Aspell dictionary.